### ISSUE TITLE
Overlapping of the tweeter's name, the tweet's age and sometimes the text of the tweet also.

### ISSUE THREAD

**Issue by rquadling** [cezarsa/silver_bird#99]
Hi.

Quite often, the tweeter's name, the age of the tweet and some of the tweet itself all overlap in the right hand corner of the tweet.

See image : https://docs.google.com/leaf?id=0B-PGIN1W94Q2NDhiZjVhZDktMGZhNi00OTYxLTlmNWMtOTgzN2Q5YmZkZWEz&hl=en

Regards,

Richard Quadling.

**Comment by pmbarros** [comment]
I've seen that happen only in the Darkness theme, and only when you select to show either the username or the full name of the sender (doesn't happen when you select to show both).

**Comment by rquadling** [comment]
Using both username and fullname works fine, but in looking at popup.html, I can't see what is so special about having them separate. I can't see anything wrong in the HTML/JS, so that leaves CSS. I think the footer's absolute positioning is causing the problem, but I can't seem to fix it.

**Comment by cezarsa** [comment]
I think I've solved it now with commit ef690a8025becc1c244b1546188dd691e7d0244d
The problem was that only the darkness sets the whole text area as a floating div. And as the user's name <a> element uses "display: inline" by default it was supposed to overlap all the time. What puzzles me is why it was only happening with short names or when the popup window was too large in width.
So, the fix was a simple one, I just set "display: block" in this element.
